tests/integration/istioio/trafficmanagement/trafficshifting/scripts/setup.txt

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+# Create a new namespace for testing purposes and enable
+# automatic Istio sidecar injection
+kubectl create namespace istio-io-tcp-traffic-shifting
```

Comment:
Please open a pr to update the docs with this new namespace.

Reply:
I raised a PR - https://github.com/istio/istio.io/pull/6368 to update documentation. Please review.